AzureCalling/app/src/main/java/com/azure/samples/communication/calling/activities/SetupActivity.java

```diff
@@ -164,21 +166,46 @@ protected void onDestroy() {
         super.onDestroy();
     }
 
+    protected void onResume() {
```

Comment:
For new methods in this PR, let's make sure we order them
1. public
2. protected
3. private

For existing methods, we can create a new PR to reorder.

Reply:
ok